reservation_units/migrations/0072_separate_pricing_model.py

```diff
@@ -0,0 +1,54 @@
+            begins = begins,
+            pricing_type = ru.pricing_type,
+            price_unit = ru.price_unit,
+            lowest_price = ru.lowest_price,
+            highest_price = ru.highest_price,
+            status = "active",
+            reservation_unit = ru,
+            tax_percentage = ru.tax_percentage,
+            
+        )
+        ru.active_pricing = active_pricing
+        ru.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reservation_units', '0071_remove_reservationunit_reservation_purposes'),
+    ]
+
+    operations = [
+        migrations.RunSQL('SET CONSTRAINTS ALL IMMEDIATE', reverse_sql=migrations.RunSQL.noop),
+        migrations.CreateModel(
+            name='ReservationUnitPricing',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('begins', models.DateField(help_text='When pricing is activated', verbose_name='Date when price is activated')),
+                ('pricing_type', models.CharField(blank=True, choices=[('paid', 'Paid'), ('free', 'Free')], help_text='What kind of pricing types are available with this reservation unit.', max_length=20, null=True, verbose_name='Pricing type')),
+                ('price_unit', models.CharField(choices=[('per_15_mins', 'per 15 minutes'), ('per_30_mins', 'per 30 minutes'), ('per_hour', 'per hour'), ('per_half_day', 'per half a day'), ('per_day', 'per day'), ('per_week', 'per week'), ('fixed', 'fixed')], default='per_hour', help_text='Unit of the price', max_length=20, verbose_name='Price unit')),
+                ('lowest_price', models.DecimalField(decimal_places=2, default=0, help_text='Minimum price of the reservation unit', max_digits=10, verbose_name='Lowest price')),
+                ('highest_price', models.DecimalField(decimal_places=2, default=0, help_text='Maximum price of the reservation unit', max_digits=10, verbose_name='Highest price')),
+                ('status', models.CharField(choices=[('past', 'past'), ('active', 'active'), ('future', 'future')], help_text='Status of the pricing', max_length=20, verbose_name='Status')),
+                ('reservation_unit', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='pricings', to='reservation_units.reservationunit', verbose_name='Reservation unit')),
+                ('tax_percentage', models.ForeignKey(default=reservation_units.models.get_default_tax_percentage, help_text='The percentage of tax included in the price', on_delete=django.db.models.deletion.PROTECT, related_name='reservation_unit_pricings', to='reservation_units.taxpercentage', verbose_name='Tax percentage')),
+            ],
+        ),
+        migrations.RunPython(migrate_active_pricing, migrations.RunPython.noop),
+        migrations.RunSQL(migrations.RunSQL.noop, reverse_sql='SET CONSTRAINTS ALL IMMEDIATE'),
```

Comment:
Why this line?

Reply:
Ooops from copy-pasting code 😄 
I needed that in one of the previous change to allow smooth rollback but it is not needed here. Removed 👍